### PR TITLE
[Method Reference 3]LPS-173756 Replace method reference with hard reference in class SoapExtender

### PR DESCRIPTION
**Pull request by W2jx**
resend

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/4902

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found

File: readme/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -1470,4 +1470,29 @@ This removal of the configuration has no replacement.
...
+
+Custom `SoapDescriptor` is no longer supported.
+
+### Who is affected?
+
+This affects anyone implementing interfaces `SoapDescriptorBuilder` and `SoapDescriptor`.
+
+### How should I update my code?
+
+- Only use `soap.endpoint.name` as the key for `soap endpoint name`.
```
Comment (dantewang) [inline readme/BREAKING_CHANGES.markdown:1491]: @W2jx Hmmmm, this is confusing. What does this mean?

Basically "how should I update" part doesn't need to always provide a way to migrate their code relying on the changed feature. Here we remove the support of replacing the `SoapDescriptor(Builder)`, so the customer can no longer use that.

- What can the customers do to update the code besides removing their custom `SoapDescriptorBuilder`?
- If they need to configure something, how do they do that in control panel?
- And does that configuration fulfill what they originally want to do? If not, we need to state that.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by W2jx** [comment]
Forced push to modify `BREAKING_CHANGES.markdown`, no logic changes.

**Comment by W2jx** [comment]
no conflicts(7/5 13:57)

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/4931

**Pull request closed without merging**